health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/configs/UICustomizations.js

```
@@ -118,6 +118,9 @@
                 onToggle={toggle}
               />):(<>{t("CHECKLIST_TOBE_CONFIGURED")}</>)
             );
+        case "CHECKLIST_LAST_UPDATE":
+          const lastModDate = row?.ServiceRequest.length !== 0 ? row?.ServiceRequest?.[0]?.auditDetails?.lastModifiedTime : row?.auditDetails?.lastModifiedTime;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)